lib/TransactionRunner.js

```diff
@@ -350,6 +350,11 @@ class TransactionRunner {
     } else {
       results = transaction.test.results = transaction.results;
```

Comment:
I thought we renamed `.results` to `.errors` so the lines above should get updated. Then perhaps you won't need the added lines below.

Reply:
Not quite. We have renamed this:

```diff
- transaction.results.general.results
+ transaction.errors
```

`transaction.results` still exists and stores Gavel validation result as-is.